dapps/marketplace/README.md

```diff
@@ -20,3 +20,39 @@ To observe the tests running in the browser:
     npm run test:browser
 
 For more information please see the README in the `test` sub-directory.
+
+## Setting up Telegram Attestation on local
+1. Go to [`@BotFather` on Telegram](https://web.telegram.org/#/im?p=@BotFather)
+2. Send the following command to create a new bot
+    ```
+    /newbot
+    ```
+    Give a name and username when prompted. Note down the Bot Token once the bot is created.
+3. Enter the command `/mybots` to list all your bots and select the bot you created in the previous step.
+4. Whitelist your IP by choosing `Bot Settings` > `Domain` > `Edit Domain` and then enter your IP address with `http://` prefix
+    ```
+    http://<YOUR IP ADDRESS>
+    ```
+5. Add the following env variables to DApp's ENVKEY or export it on the terminal
+    ```
+    TELEGRAM_BOT_USERNAME=<YOUR BOT USERNAME>
+    ```
+5. Add the following env variables to bridge server's ENVKEY or export it on the terminal
+    ```
+    TELEGRAM_BOT_TOKEN=<YOUR BOT TOKEN HERE>
+    ```
+6. Start the DApp with the following command
+    ```
+    HOST=<YOUR IP ADDRESS> PORT=80 yarn start
+    ```
+
+### Additional step for Telegram Webhooks
+When you want to run the Telegram Webhooks for group join event verification, you should do the following. 
+1. Create a web tunnel for bridge server. [Refer `@origin/bridge`'s README.md](https://github.com/OriginProtocol/origin/blob/master/infra/bridge/README.md#setup-tunnel-for-webhooks-optional)
+2. Add the bot you created to the group as an admin. 
```

Comment:
Very cool readme 👏 

Can you also add that you need to make the group public. And there should probably be some configuration somewhere where you specify the name of the group that dapp needs to open.